utilities/hamlet/Dockerfile

```diff
@@ -1,4 +1,4 @@
-ARG BASE_IMAGE=hamletio/hamlet:latest
+ARG BASE_IMAGE=hamletio/engine:latest
```

Comment:
This ARG is based on the images in the docker repo rather than the local one like you would have with a multi stage pipeline. 

Might be better that we actually push the base image we throw away in the jenkisfile build and this references that. 

So that would change this to BASE_IMAGE=hamletio/base:latest which is just out base image with the depdenceis

Reply:
rightyo, will update now